Assets/Scripts/PlayerScripts/PlayerMovement.cs

```diff
@@ -278,6 +241,16 @@ public void TriggerJump(InputAction.CallbackContext context)
             }
         }
     }
+
+    public void TriggerStomp(InputAction.CallbackContext context)
+    {
+        if (!_playerInputEnabled) return;
+
+        if (!context.canceled){
```

Comment:
Will the stomp sound and stomp method be triggered twice? (Once for context.started and once for context.performed?)

Reply:
From my test, checking started and performed doesn't actually prevent it from triggering twice (once for press once for release), only checking cancelled worked

Reply:
Did you hear that it triggered twice?